MAUI/Calendar/migration.md

```diff
@@ -18,8 +18,8 @@ To make the migration from the [Xamarin SfCalendar](https://www.syncfusion.com/x
 <th>Xamarin SfCalendar</th>
 <th>.NET MAUI SfCalendar</th></tr>
 <tr>
-<td>Syncfusion.SfCalendar.XForms</td>
-<td>Syncfusion.Maui.SfCalendar</td></tr>
+<td>Syncfusion.Calendar.XForms</td>
```

Comment:
Syncfusion.SfCalendar.XForms - Check the namespace

Reply:
Checked and changed the namespace to Syncfusion.SfCalendar.XForms